tinyformat.h

```diff
@@ -846,6 +854,7 @@ class FormatList
                             const FormatList& list);
 
     private:
+        FormatList();
```

Comment:
Why is this necessary?

Reply:
Good question actually.  @nigels-com why did you add this?

Reply:
A FormatList default constructor looks like another place that ought to be doing initialisation.

Reply:
The default constructor is deleted because a nontrivial constructor was declared, so this should be unncessary.  I'll remove it.

Reply:
Better to be explicit, IMHO.